cl/alerts/tasks.py

```diff
@@ -244,3 +309,76 @@ def send_unsubscription_confirmation(
     html = html_template.render(email_context)
     msg.attach_alternative(html, "text/html")
     msg.send()
+
+
+@app.task()
+def send_docket_to_webhook(
+    d_pk: int,
+    since: datetime,
+    webhook_recipients: list[str],
+) -> None:
+    """POSTS the DocketAlert to the recipients webhook(s)
+
+    :param d_pk: The Docket primary key
+    :param since: Start time for querying the docket entries
+    :webhook_recipients: The list of email recipients to send the webhook to
+    :return: None
+    """
+
+    docket_entries = DocketEntry.objects.filter(
+        date_created__gte=since, docket_id=d_pk
+    )
+    if docket_entries.count() == 0:
+        # No new docket entries.
+        return
+
+    webhooks = Webhook.objects.filter(
+        event_type=WebhookEventType.DOCKET_ALERT,
+        user__email__in=webhook_recipients,
+        enabled=True,
+    )
+
+    serialized_docket_entries = []
+    for de in docket_entries:
+        serialized_docket_entries.append(DocketEntrySerializer(de).data)
+
+    for webhook in webhooks:
+        post_content = {
+            "webhook": {
+                "event_type": webhook.event_type,
+                "version": webhook.version,
+                "date_created": webhook.date_created.isoformat(),
```

Comment:
Don't forget to do the `deprecation_date` field (or similar) here.

Reply:
I've added it. Now, is set `None`.